bin/phpspec

```diff
@@ -1,26 +1,28 @@
 #!/usr/bin/env php
 <?php
 
-define('PHPSPEC_VERSION', '3.0.x@dev');
```

Comment:
I couldn't find any usages so would prefer to take it out. It can be added back during beta if extension writers ask - there's probably a better way for us to expose version though I think?